inc/command/class-command.php

```diff
@@ -445,6 +451,70 @@ protected function codecept( InputInterface $input, OutputInterface $output ) {
 		return $return;
 	}
 
+	/**
+	 * Bootstraps configuration files.
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 *
+	 * @return int
+	 */
+	protected function bootstrap( InputInterface $input, OutputInterface $output ) : int {
+		$configs = [ 'codespaces' ];
+
+		$options = $input->getArgument( 'options' );
+		$subsubcommand = $options[0] ?? null;
+
+		if ( empty( $subsubcommand ) ) {
+			$output->writeln( '<warning>You need to select a configuration set to bootstrap/generate.</warning>' );
+			$output->writeln( sprintf( 'Available configuration sets are: %s.', implode( ', ', $configs ) ) );
+			$output->writeln( 'eg: composer dev-tools bootstrap codespaces' );
+
+			return 1;
+		}
+
+		if ( ! in_array( $subsubcommand, $configs, true ) ) {
+			$output->writeln( sprintf( '<error>Could not find the target configuration set generator for "%s".</error>', $subsubcommand ) );
+			return 1;
+		}
+
+		call_user_func( [ $this, 'bootstrap_' . $subsubcommand ], $input, $output );
```

Comment:
You could also do something like:
```suggestion
		$method = "bootstrap_{$subsubcommand}";
		$this->$method( $input, $output );
```

Reply:
By the way, the curly braces are usually not needed in a interpolated string unless followed by a character that could be considered part of the variable.
so: `$method = "bootstrap_$subsubcommand";`

Reply:
@roborourke I'd prefer using `call_user_func` as it seems more readable to me, would you agree ? is the requested change more consistent with how other bits of the code looks like ?

@mikelittle re the braces, I'd opt for consistency when it's low effort like this as a general preference. eg: if we have two instances of interpolated strings, I'd want them to look consistent, eg: `something_{$var}` and `something_{$var}else`. Also looks specially readable if you don't have syntax highlighting on.

Reply:
Yeah that’s fine to use call_user_func, I don’t think there’s any precedent in this module for it